src/grid/onedgrid.py

```diff
@@ -58,7 +58,7 @@ def GaussLaguerre(npoints, alpha=0):
         raise ValueError(f"Alpha need to be bigger than -1, given {alpha}")
     points, weights = roots_genlaguerre(npoints, alpha)
     weights = weights * np.exp(points) * np.power(points, -alpha)
-    return OneDGrid(points, weights)
+    return OneDGrid(points, weights, (0, np.inf))
 
 
 def GaussLegendre(npoints):
```

Comment:
I agree but I will differ this to another PR. I rather not choose a name by myself, so it would be good to have a meeting at a later time to decide how we want the naming API to change.

Reply:
good and I agree that cleanups that do not change behavior are better done in separate PRs, to make reviewing easier. Good point.